roles/iota-prom-exporter/tasks/deps.yml

```
@@ -0,0 +1,10 @@
+- name: Install some packages
```

Rename to specific package type might be better
```suggestion
- name: Install dependency packages
```

---

Fixed.